src/components/Resources/index.js

```diff
@@ -1,28 +1,11 @@
 import React, { useState, useEffect } from "react";
 import axios from "axios";
-import { makeStyles } from "@material-ui/core/styles";
-import clsx from "clsx";
 import PersonalMenu from "../PersonalMenu";
-import Card from "@material-ui/core/Card";
-import CardHeader from "@material-ui/core/CardHeader";
-import CardContent from "@material-ui/core/CardContent";
-import CardActions from "@material-ui/core/CardActions";
-import Collapse from "@material-ui/core/Collapse";
-import Avatar from "@material-ui/core/Avatar";
-import IconButton from "@material-ui/core/IconButton";
-import Rating from "@material-ui/lab/Rating";
-import StarBorderIcon from "@material-ui/icons/StarBorder";
-import Typography from "@material-ui/core/Typography";
-import { red } from "@material-ui/core/colors";
-import BookmarkIcon from "@material-ui/icons/Bookmark";
-import ShareIcon from "@material-ui/icons/Share";
-import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
-import MoreVertIcon from "@material-ui/icons/MoreVert";
-import Grid from "@material-ui/core/Grid";
-import TextareaAutosize from "@material-ui/core/TextareaAutosize";
-import { Button } from "@material-ui/core";
-import LinkIcon from "@material-ui/icons/Link";
 import Search from "../Search";
+import Grid from "@material-ui/core/Grid";
+import { red } from "@material-ui/core/colors";
+import { makeStyles } from "@material-ui/core/styles";
+import { ResourceCard } from "./ResourceCard";
 
 const useStyles = makeStyles(theme => ({
```

Comment:
We can probably remove this `useStyles` block here, right? (Since it's been moved to the `ResourcesCard` component)

Reply:
Just pushed my changes to remove the unused styling. I think this was added when you merge this branch with master earlier. 😅